receiver/vmmetricsreceiver/vm_metrics_collector.go

```diff
@@ -127,7 +136,14 @@ func (vmc *VMMetricsCollector) scrape() {
 			mSystemCPUSeconds.M(cpuStat.System),
 			mIdleCPUSeconds.M(cpuStat.Idle),
 			mIowaitCPUSeconds.M(cpuStat.Iowait))
+		if prevStat != nil {
+			stats.Record(ctx, mProcessesCreated.M(int64(stat.ProcessCreated - prevStat.ProcessCreated)))
```

Comment:
Should `ProcessRunning` and `ProcessBlocked` be updated too?

Reply:
No. I checked, they are instantaneous values.